src/service/navigation.js

```diff
@@ -458,6 +459,19 @@ export class Navigation {
         if (targetAttr != '_top' && targetAttr != '_blank') {
           target.setAttribute('target', '_blank');
         }
+        return; // bail early.
+      }
+
+      // Accessibility fix for IE browser
```

Comment:
Is there a blog post or similar that this could point to? Otherwise please explain what this fixes and how it works.

Reply:
@cramforce https://github.com/ampproject/amphtml/issues/18671 has some discussion. tl;dr: a common plugin in WP needed since IE does not move the a11y cursor when doing anchor navigation.

Reply:
I understand why it is needed. But would be nice to have a reference in the code for future code readers.

Reply:
Should I add a comment in this branch? ( It is merged )

Reply:
Would need to be a separate PR. Thank you!

Reply:
@aghassemi 
PR: https://github.com/ampproject/amphtml/pull/19064